src/styles/global/ie11-fixes.styl

```diff
@@ -0,0 +1,6 @@
+html
+body
+#app-container
+#app-container > div
+  height 100%      //Enables display: flex child items to grow properly in IE11.
+  min-height 100%  //Prevents the above fix from limiting the height of said elements in other browsers.
```

Comment:
@rogerhutchings These styles are applied normally across all browsers; (no Modernizr, etc) the fixes were separated into an independent ie11-fixes.styl file only for our labelling purposes.

@simoneduca Let me check in with you regarding removing body.styl; it's likely that I need to re-align myself with how everyone's planning to organise the CSS files.